apps/src/templates/manageStudents/NoSectionCodeDialog.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import Button from '../Button';
+import {SectionLoginType} from '@cdo/apps/util/sharedConstants';
+
+const styles = {
+  buttonStyle: {
+    float: 'right'
+  }
+};
+
+export default class NoSectionCodeDialog extends React.Component {
+  static propTypes = {
+    typeClassroom: PropTypes.string,
+    handleClose: PropTypes.func
+  };
+
+  getClassroomType = () => {
```

Comment:
We're starting to move away from using arrow functions in class properties. [Full Reasoning](https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1). [TL;DR](https://github.com/airbnb/javascript/tree/master/react#methods). The preferred way is to instead bind these in the constructor (see example in the TL;DR link).